### DISCUSSION TITLE
Why is Cloudant not recommended?

### DISCUSSION THREAD

**Discussion by moshin34** [vrtmrz/obsidian-livesync#393] [Q&A]
_"We are still able to use IBM Cloudant. However, it is not recommended for several reasons nowadays."_

What are the reasons exactly?

**Comment by vrtmrz** [comment]
Thank you for asking me!
There are two reasons.

1. Difficulties in creating an account
In some cases, our credit cards were denied. After some trying, in my case, disabling 3D Secure on the credit card worked, but I do not know if this is the right solution.
2. Free-tier Limitation
The free plan limits the interval between requests and the size of one document. This was a major limitation in previous versions of Self-hosted LiveSync.

For now, these limitations can be resolved with some configurations. However, I felt these configurations were too complex, so I decided that it was not recommended.

If it could be easier, IBM Cloudant would return to the mainstream again.
